src/events/render-advert.ts

```diff
@@ -200,6 +201,29 @@ const renderAdvert = (
 	advert: Advert,
 	slotRenderEndedEvent: googletag.events.SlotRenderEndedEvent,
 ): Promise<boolean> => {
+	const matchingAd = window.guardian.a9WinningBids?.find(
+		(bidResponse) => bidResponse.slotID == advert.id,
+	);
+
+	const isA9GumGum = matchingAd?.amznp === '1lsxjb4';
+
+	if (
+		slotRenderEndedEvent.advertiserId != null &&
+		slotRenderEndedEvent.advertiserId === 4751525411 &&
+		isA9GumGum
+	) {
+		const adSlotId = advert.node.id;
+		logGumGumWinningBid(
+			adSlotId,
+			slotRenderEndedEvent.advertiserId.toString(),
+		);
+	}
+
+	window.addEventListener('beforeunload', () => {
+		console.log(`User is clearing data in the browser`);
+		window.guardian.a9WinningBids = [];
+	});
```

Comment:
what's this for?

Reply:
the window object doesn't persist across page loads.

Reply:
ok great so safe to remove this Im assuming 👍